src/config.rs

```diff
@@ -262,6 +262,10 @@ macro_rules! cf_config {
             pub compaction_guard_min_output_file_size: ReadableSize,
             #[config(skip)]
             pub compaction_guard_max_output_file_size: ReadableSize,
+            #[config(skip)]
+            pub zstd_dict_size: i32,
+            #[config(skip)]
+            pub zstd_sample_size: i32,
```

Comment:
let's rename these two config to `bottommost_compression_zstd_dict_size` and `bottommost_compression_zstd_sample_size` respectively. Let's leave them hidden and not mentioning them in config-template.toml.